### PR TITLE
View question

### PR DESCRIPTION
**Pull request by ljjb97**
Questions which have been added to a survey will now be answerable when viewing a specific survey. On the submission of the survey a SurveyResponse object is saved to the DB.

### CODE REVIEW

File: src/main/resources/templates/numberquestion.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Number Question</title>
+</head>
+<body>
+<form>
+    <div th:fragment="num">
+        <div th:text="${question.getDescription()}"></div>
+        <input type="number" th:min="${question.getMinRange()}" th:max="${question.getMaxRange()}"/>
```
Comment (Aleksandar-Veselinovic) [inline src/main/resources/templates/numberquestion.html:12]: Is this how number questions should look like? Maybe it should be a range?

File: src/main/resources/templates/textquestion.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Text Question</title>
+</head>
+<body>
+<form>
+    <div th:fragment="text">
+        <div th:text="${question.getDescription()}"></div>
+        <input type="text" th:maxlength="${question.getCharLimit()}"/>
```
Comment (Aleksandar-Veselinovic) [inline src/main/resources/templates/textquestion.html:12]: Would it be better to use a text area input here?

File: src/main/resources/templates/numberquestion.html (hunk 1 above)
Comment (mralifahd) [reply src/main/resources/templates/numberquestion.html:12]: Yea just to add to this I think <input type="range"> will be good for our ui here

File: src/main/resources/templates/textquestion.html (hunk 2 above)
Comment (ljjb97) [reply src/main/resources/templates/textquestion.html]: Done! Changed to textarea element

File: src/main/resources/templates/numberquestion.html (hunk 1 above)
Comment (ljjb97) [reply src/main/resources/templates/numberquestion.html]: Done! Changed to a range type of input!

Comment (Aleksandar-Veselinovic) [reply src/main/resources/templates/numberquestion.html]: does the input show the min and max values

File: src/test/java/com/example/SurveyControllerTest.java [hunk 3]
```diff
@@ -93,4 +95,22 @@ public void testAddQuestion() throws Exception {
...
+        s.setDescription("test");
+        ArrayList<SurveyQuestion> questions = new ArrayList<>();
+        TextQuestion txtQ = new TextQuestion(500);
+        txtQ.setDescription("Do you like ice cream?");
+        questions.add(txtQ);
+        s.setQuestions(questions);
+
+        mockMvc.perform(get("/survey?selectedSurvey=1"))
+                .andDo(print())
+                .andExpect(MockMvcResultMatchers.model().attribute("survey",
```
Comment (Bnortron) [inline src/test/java/com/example/SurveyControllerTest.java:112]: Should there also be a check for the view name here?

**Review by Bnortron — APPROVED** [review]
Functionality & testing look good to merge

**Pull request merged**